RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+Reducer.swift

```diff
@@ -11,6 +11,7 @@ public struct NonFungibleTokenDetails: Sendable, FeatureReducer {
 		public let ledgerState: AtLedgerState
 		public let stakeClaim: OnLedgerEntitiesClient.StakeClaim?
 		public let isClaimStakeEnabled: Bool
+		var hideAsset: HideAsset.State?
```

Comment:
I suppose this should also be public?

Reply:
nothing needs to be public actually :)

Reply:
Sure, but since the struct and all the other properties are public, this one should be as well. There is no reason for them to have different access levels, it only causes confusion.

Reply:
IMO the confusion is caused by leaving the others public, and what we should do is actually remove the access modifier from thise.

Reply:
Previously we had separate modules for every feature, so everything had to be public. Now there is no technical reason for it, but we should at the very least be consistent within a feature.